_topic_maps/_topic_map.yml

```diff
@@ -405,10 +405,13 @@ Dir: post_installation_configuration
 Distros: openshift-origin,openshift-enterprise
 Topics:
 - Name: Post-installation configuration overview
+  Distros: openshift-enterprise
   File: index
 - Name: Configuring a private cluster
   Distros: openshift-enterprise,openshift-origin
   File: configuring-private-cluster
+- Name: Bare metal configuration
```

Comment:
Should be baremetal

Reply:
According to the IBM style guide, it's two words. It's pretty consistently that way throughout our documentation.

Reply:
OK, sounds good.